invokeai/app/api/routers/models.py

```diff
@@ -116,36 +80,50 @@ async def update_model(
     responses= {
```

Comment:
If the `update` route is a PATCH request at `/`, does it make sense to have this `create` route be a POST request at `/`?

Reply:
Yes! The only reason I chose `/import` was that I was already POSTing to `/` for the model update. But now that's been changed to PATCH, so `/` is OK.